rpm-build/SPECS/securedrop-workstation-dom0-config.spec

```diff
@@ -15,13 +15,13 @@
 
 Name:		%{srcname}
 Version:	%{version}
-Release:	1%{?dist}
+Release:	0.rc1.1%{?dist}
```

Comment:
It seems like this is how we've always been doing it, but I think the correct way is:
```
%global version 0.8.0~rc1
Release: 1%{?dist}
```

(see https://rpm-packaging-guide.github.io/#what-is-a-spec-file where Version is "The upstream version number of the software." and Release is just a plain integer)

I'll file a bug, not something to block on